Assets/Scripts/UI/Pickup Pup/Core/PPUIController.cs

```diff
@@ -41,17 +57,46 @@ public void LoadHome()
 		sceneController.LoadHome();
 	}
 
```

Comment:
I don't see (and maybe it's just GitHub not showing me) any `#endregion` directive to match the `#region MonoBehaviourExtendedOverrides`